package.json

```diff
@@ -162,6 +162,7 @@
     "@o3r/dev-tools": "workspace:^",
     "@o3r/eslint-config-otter": "workspace:^",
     "@o3r/eslint-plugin": "workspace:^",
+    "@o3r/test-helpers": "workspace:^",
```

Comment:
maybe you need to add the build of this package in `build:tools` script

Reply:
it's actually no longer needed